CODEOWNERS

```diff
@@ -464,6 +464,7 @@ Makefile* @cilium/build
 /pkg/aws/ @cilium/aws
 /pkg/azure/ @cilium/azure
 /pkg/backoff/ @cilium/sig-agent
+/pkg/bufuuid/ @cilium/sig-agent
```

Comment:
hmm sig-agent is kind of general; I guess there's no perf team willing to own this?

Reply:
One possibility could be the `cilium/sig-scalability` team, which is currently owning:

```
CODEOWNERS:# - @cilium/sig-scalability:
CODEOWNERS:/.github/actions/cl2-modules/ @cilium/sig-scalability
CODEOWNERS:/.github/workflows/*scale*.yaml @cilium/sig-scalability @cilium/github-sec @cilium/ci-structure
CODEOWNERS:/.github/workflows/*perf*.yaml @cilium/sig-scalability @cilium/github-sec @cilium/ci-structure
CODEOWNERS:/operator/pkg/ciliumendpointslice @cilium/sig-scalability
```

I wasn't totally sure honestly, given that many other generic packages are owned by `sig-agent`.

Reply:
(sorry for the delay, vacation)
@marseel do you feel like scalability should own this?

Reply:
sure, makes sense.